exercises/mark-griffith/answers.js

```diff
@@ -0,0 +1,135 @@
+// Question 1
+function greet(greet) {
+  return `Good morning, ${greet}!`;
+}
+
+// Question 2
+function logOddNumbers() {
+    let num = 1;
+    while(num<=99){
+      if(num % 2 == 0){
+        num++;
+      }
+      else{
+        console.log(num);
+        num++;
+      }
+  }
+}
+
+// Question 3
+function area(length, width) {
+    return length * width;
+}
+
+// Question 4
+function otherAngle(first, second) {
+    let total= 180;
+    total -= first + second;
+    if(total >= 1){
+      return total;
+    }
+    else{
+      console.log("No");
+    }
+}
+
+// Question 5
+function diameterFromArea(area) {
+  return 2 * Math.sqrt((area/Math.PI));
+}
+
+// Question 6
+function totalPrice(price,percent) {
+  percent = percent/100;
+  percent = price * percent;
+  price = price + percent;
+  return price;
+}
+
+// Question 7
+function shortLongShort(str1,str2) {
+  let string = "";
+   if(str1.length > str2.length){
+     string = str2 + str1 + str2;
+     return string;
+   }
+   else{
+     string = str1 + str2 + str1;
+     return string;
+   }
+}
+
+// Question 8
+function describeAge(age) {
+  let desc = (age <= 12) ? "You're a kid." : (age >= 13 && age <= 17) ? "You're a teenager." : (age >= 18 && age <= 64) ? "You're an adult." : "You're an elderly.";
```

Comment:
You are actually making unnecessary conditional checks here and it makes this statement too long. Think about why this works. Why does the implementation below get away with fewer checks?

```javascript
function describeAge(age) {
  return age > 65 ? "You are an elderly? " : age > 18 ? "You are a teenager?" : "You are a kid.";
}
```